tests/test_models/test_model_meta.py

```diff
@@ -127,6 +127,19 @@ def test_check_training_datasets_can_be_derived(model_meta: ModelMeta):
     model_meta.get_training_datasets()
 
 
+def test_loader_kwargs_persisted_in_metadata():
+    model = mteb.get_model(
+        "sentence-transformers/all-MiniLM-L6-v2",
+        device="cpu",
```

Comment:
Can you add this check rather then device
```suggestion
        not_existing_param=123,
```

Reply:
Changed to this, and its working. Isn't it should also worked with sentence_transformer, or is it just because that particular parameter was invalid?

Reply:
> is it just because that particular parameter was invalid?

Yes, problem with it